mlflow/server/js/src/experiment-tracking/components/experiment-page/models/ExperimentPageUIState.tsx

```diff
@@ -6,26 +6,6 @@ import type { ChartSectionConfig } from '@mlflow/mlflow/src/experiment-tracking/
 import type { RunsGroupByConfig } from '../utils/experimentPage.group-row-utils';
 import { RunsChartsLineChartXAxisType } from '../../runs-charts/components/RunsCharts.common';
 
-export const EXPERIMENT_PAGE_UI_STATE_FIELDS = [
```

Comment:
@hubertzub-db is there some reason this was initially hardcoded instead of taking `Object.keys()` from the result of `createExperimentPageUIState()` below?

Reply:
I think that at some point and for some reason we wanted to have fine grained control and selectively persist fields, hence the separate list. If the result of `Object.keys()` is equal then surely we can use it.